package.json

```diff
@@ -77,6 +77,7 @@
     "@deploysentinel/playwright": "^0.3.3",
     "@playwright/test": "^1.31.2",
     "@snaplet/copycat": "^0.3.0",
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
why are we adding a new testing library? We already have playwright

Reply:
Hi @PeerRich! We are using vitest to make the tests (a lib already installed on the project), we are just using the RTL to render the components. We can try to use only playwright for it but this lib is used to test pages instead only specifc components, but this can be more complicated